src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({person}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{person.name ? `I am ${person.age}` : null}</p>
+    <p className="Person__partner">
+      {person.isMarried ? (
+        `${person.partnerName} is my
+
+        ${person.sex === 'f' ? 'husband' : 'wife'}`)
+
+        : 'I am not married'}
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```